spec/android_device_spec.rb

```diff
@@ -1,7 +1,5 @@
+require 'spec_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.